src/app/problems/problem-set/problem-set.component.html

```diff
@@ -74,7 +74,7 @@
 
     <ng-container matColumnDef="difficulty">
         <th mat-header-cell *matHeaderCellDef mat-sort-header="difficulty">Difficulty</th>
-        <td mat-cell *matCellDef="let question"> {{question.difficulty || 'No Difficulty'}} </td>
+        <td mat-cell *matCellDef="let question"> {{question.difficulty === 'NORMAL' ? 'MEDIUM' : question.difficulty || 'No Difficulty'}} </td>
```

Comment:
To solve this properly you have to call the difficulty service and find it from there. This will break if difficulties changes.